AdyenComponents/PayTo/PayToComponent.swift

```diff
@@ -15,6 +15,29 @@ public final class PayToComponent: PaymentComponent,
         static let flowSelectionTitleLabelItem = "flowSelectionTitleLabel"
         static let flowSelectionItem = "flowSelectionSegmentedControl"
         static let continueButtonItem = "continueButton"
+        static let identifierPickerItem = "identifierPicker"
+        static let firstNameInputItem = "firstNameTextfield"
+        static let lastNameInputItem = "lastNameTextfield"
+    }
+
+    private enum AccountIdentifier: CustomStringConvertible, CaseIterable {
+        case phone
+        case email
+        case abn
+        case organizationID
+
+        public var description: String {
+            switch self {
+            case .phone:
+                return "Phone"
```

Comment:
Todo for translation for these words as well. (except ABN i think?)

Reply:
yes, just saw the thread on channel. I added this questions in my to-ask list but now I know the answer :) I gonna add TODO here as well.